### DISCUSSION TITLE
感谢作者，反馈一个问题，自从1.38.30版本后，总是出现终端连接丢失的问题

### DISCUSSION THREAD

**Discussion by zimu72** [electerm/electerm#3352] [General]
大概从1.38.30版本后，总是出现终端连接丢失的问题，我用的最新版本是1.38.60，基本上中间每个版本都试了，系统环境是Windows10和windows11，虚机用的vmware上面装的centos7，而且试过退回到1.38.30，发现退回去就没问题了
![屏幕截图 2024-04-09 111613](https://github.com/electerm/electerm/assets/28486924/88cde57b-8d67-426b-af01-590ba5bcb0f0)

**Comment by zxdong262** [comment]
1.38.65可以试试吗？

**Reply by zimu72** [reply]
试了下1.38.65可以啦，太感谢啦！
